types/json-schema-generator/index.d.ts

```diff
@@ -0,0 +1,23 @@
+// Type definitions for json-schema-generator 2.0
+// Project: https://github.com/krg7880/json-schema-generator
+// Definitions by: Marcell Toth <https://github.com/marcelltoth>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { JSONSchema4 } from 'json-schema';
+
+/*~ Note that ES6 modules cannot directly export callable functions.
+ *~ This file should be imported using the CommonJS-style:
+ *~   import x = require('someLibrary');
+ *~
+ *~ Refer to the documentation to understand common
+ *~ workarounds for this limitation of ES6 modules.
+ */
+
+/*~ This declaration specifies that the function
+ *~ is the exported object from the file
+ */
+
+export = generate;
```

Comment:
```suggestion
export = jsonToSchema;
```
if you switch to that one, everyone would understand that you are following naming convention from teh source and follow source module details:
https://github.com/krg7880/json-schema-generator/blob/27d5a84721c4cd496a2a37bbefda09c07a59f6a1/lib/index.js#L7
it's a nice touch

Reply:
Makes sense, I'll update it.